pkg/operator/controller/ingress/status_test.go

```diff
@@ -108,6 +111,131 @@ func cond(t string, status operatorv1.ConditionStatus, reason string, lt time.Ti
 	}
 }
 
+func TestComputeIngressDegradedCondition(t *testing.T) {
+	tests := []struct {
+		name           string
+		conditions     []operatorv1.OperatorCondition
+		expectedStatus operatorv1.ConditionStatus
+		expectRequeue  bool
+	}{
+		{
+			name: "not admitted",
```

Comment:
Add the empty test?

		{
			name:           "no conditions",
			expectedStatus: operatorv1.ConditionFalse,
			expectRequeue:  false,
		},

Reply:
Done.